src/uShip.Logging.Tests/LoggingEventPropertiesBuilderTests.cs

```diff
@@ -53,6 +54,20 @@ public void Should_sanitize_creditCard()
         }
 
         [Test]
+        public void Should_sanitiz_credit_cards_from_loggable_exception()
```

Comment:
I am creating a loggable exception with 
{\"user\":{\"password\":\"foobar484634198\"}

Then verifying the exception data dictionary is sanitized
{\"user\":{\"************\":\"****\"}